.codeclimate.yml

```diff
@@ -30,11 +30,11 @@ checks:
 
 plugins:
   brakeman:
-    enabled: true
+    enabled: false
```

Comment:
Are plugins enabled by default? Would it work just as well to remove these altogether?

Reply:
good question, I'll give it a go

Reply:
seems as though plugins are not enabled by default, so I've removed all disabled ones (and also removed pep8 since we have no python files in the repo)